Sources/BluetoothMessageProtocol/Characteristic/SIG/CharacteristicCyclingSpeedCadence.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  File.swift
+//  
+//
+//  Created by Joe Blau on 9/10/20.
+//
+
+import Foundation
+import DataDecoder
+import FitnessUnits
+
+/// BLE Alert Level Characteristic
+///
+/// The CSC Measurement characteristic (CSC refers to Cycling Speed and Cadence) is a variable length structure containing a Flags field and, based on the contents of the Flags field, may contain one or more additional fields as shown in the tables below.
+@available(swift 3.1)
+@available(iOS 10.0, tvOS 10.0, watchOS 3.0, OSX 10.12, *)
+final public class CharacteristicCyclingSpeedCadence: Characteristic {
+    
+    /// Characteristic Name
+    public static var name: String { " Cycling Speed and Cadence" }
+    
+    /// Characteristic UUID
+    public static var uuidString: String { "2A5B" }
+    
+    
+    /// Name of the Characteristic
+    public var name: String { Self.name }
+    
+    /// Characteristic UUID String
+    public var uuidString: String { Self.uuidString }
+    
+    
+    public enum Flags: UInt8 {
+        case wheelPresent = 0
+        case crankPresent = 1
+    }
+    
+    private(set) public var flags: Flags
+        
+    public enum PresentData {
```

Comment:
I would rather see this as a actual Struct that is: Revolutions + lastEvent Time
and then have one var for wheel and one for crank